src/EnergyPlus/DataHeatBalance.hh

```diff
@@ -690,6 +690,7 @@ namespace DataHeatBalance {
         bool anySurfacesWithSpace = false;    // True if any surfaces in a zone have a space assigned in input
         EPVector<int> spaceIndexes;           // Indexes to spaces in this zone
         int numSpaces = 0;                    // Number of spaces in this zone
+        bool noEMPDorHAMT = false;            // True if surface heat balance uses EMPD or HAMT
```

Comment:
This should be initialized to `true`. So why didn't this cause diffs? Do those calcs even get used for anything?

Reply:
I saw that and fixed it. I am confused at why this did not cause more problems. The initial Array1D<bool>no_ht_EMPD_or_HAMT was never initialized and just happened to be true. This is used to reset humidity variables in 2 places in ZoneTempPredictorCorrector. Only the unit test identified a problem with my implementation, but why didn't this cause more diffs is a good question.

        if (no_ht_EMPD_or_HAMT) {
            state.dataHeatBalFanSys->SumHmARaW(ZoneNum) = 0.0;
            state.dataHeatBalFanSys->SumHmARa(ZoneNum) = 0.0;
        }